cxx-squid/src/test/resources/visitors/binary_logical.cc

```diff
@@ -0,0 +1,7 @@
+int main(int argc, char* argv[])
+{
+   if ((argc == 1) &&         // +1
+       (argv[0] != NULL)) {   // +1
```

Comment:
Not sure if this should not be 1 only? Maybe you can add also the other samples from PDF section "Sequences of logical operators" to verify it.

Reply:
Good catch, this example is actually correct (though not so helpfully annotated, should be +1 for if, and +1 for &&), but I hadn't read the 'sequence of logical operators' section so the other samples were incorrect.